scripts/x-tree-view.exports.json

```diff
@@ -23,6 +23,21 @@
   { "name": "SimpleTreeViewSlots", "kind": "Interface" },
   { "name": "SingleSelectTreeViewProps", "kind": "TypeAlias" },
   { "name": "TreeItem", "kind": "Variable" },
+  { "name": "TreeItem2", "kind": "Variable" },
+  { "name": "TreeItem2Content", "kind": "Variable" },
+  { "name": "TreeItem2Icon", "kind": "Function" },
+  { "name": "TreeItem2IconContainer", "kind": "Variable" },
+  { "name": "TreeItem2IconProps", "kind": "Interface" },
+  { "name": "TreeItem2IconSlotProps", "kind": "Interface" },
+  { "name": "TreeItem2IconSlots", "kind": "Interface" },
+  { "name": "TreeItem2Label", "kind": "Variable" },
+  { "name": "TreeItem2Props", "kind": "Interface" },
+  { "name": "TreeItem2Provider", "kind": "Function" },
+  { "name": "TreeItem2ProviderProps", "kind": "Interface" },
+  { "name": "TreeItem2Root", "kind": "Variable" },
+  { "name": "TreeItem2SlotProps", "kind": "Interface" },
+  { "name": "TreeItem2Slots", "kind": "Interface" },
+  { "name": "TreeItem2TransitionGroup", "kind": "Variable" },
```

Comment:
So, is it `GroupTransition` or `TransitionGroup`? 😄 🙈

Reply:
:see_no_evil: